docs/tutorials/pachyderm-cat-dog.rst

```diff
@@ -23,8 +22,8 @@ cluster, and then use the model in an inferencing pipeline.
 ************
 
 After completing the steps in this tutorial, you will have a a fully-built batch inferencing
-`pipeline <https://docs.pachyderm.com/latest/learn/glossary/pipeline/>`__ with results and you will
```

Comment:
see above comment

Reply:
leaving this one out because of some upcoming changes